### ISSUE TITLE
kaggle_dac_loss_accuracy_plots.png

### ISSUE THREAD

**Issue by jzhang82119** [facebookresearch/dlrm#86]
is this plot generated during the training and testing?
Can the plot source code be shared as well?

**Comment by mnaumovfb** [comment]
The plots are generated during training. Notice that we keep track of training loss/accuracy and also periodically evaluate the model on a test set, which gives rise to the test loss/accuracy curve.

The plotting script should be simple to write, by extracting and plotting the printed values of train and test loss/accuracy. However, I don't have the bandwidth to push it out right now.
